05.ls/ls_1.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+COLUMN_COUNT = 3
+
+def main
+  files = Dir.glob('*')
+
+  transposed_files = make_transposed_files(files)
+
+  max_file_size = files.map(&:size).max
+  output_files(transposed_files, max_file_size)
+end
+
+def make_transposed_files(files)
```

Comment:
「オブジェクトを作る」というときの「作る」はmakeよりもbuildの方がよく使われる印象です（なぜかはわかりませんが・・・）。
なので

```suggestion
def build_transposed_files(files)
```

の方がベターかもしれません。

Reply:
コメントありがとうございます。build と修正しました。